mantidproject/mantid
# Fix bug with underscore in table column names

Fixes the issue with blank plots being loaded when created from a table window.

**To test:**

- Follow the instructions in the bug description in the issue report.
- Apply the fix and repeat instructions. Should appear fixed now.

Fixes #18196 .

---

#### Reviewer ####

Please comment on the following ([full description](http://www.mantidproject.org/Individual_Ticket_Testing)):

##### Code Review #####


##### Functional Tests #####


- Does everything look good? Comment with the ship it emoji but don't merge. A member of `@mantidproject/gatekeepers` will take care of it.
